create.html

```diff
@@ -7,30 +7,32 @@
     <title>Quiz App / create</title>
     <link rel="stylesheet" href="./styles/index.css" />
   </head>
-  <body>
-    <div class="app-grid">
+  <body class="create">
+    <div class="app-grid text-center">
       <h1>Create a Question</h1>
-      <form action="" class="app-grid__scrollarea h-100vh" autocomplete="off">
-        <section>
+      <form action="" class="app-grid__scrollarea h-100vh " autocomplete="off">
+        <section class="create__section-wrap">
           <label for="add-question">Add question</label>
-          <input id="add-question" type="text" />
+          <textarea id="add-question" type="text" name="add-question" cols="35" rows="7" ></textarea>
         </section>
-        <section>
+        <section class="create__section-wrap">
           <label for="add-answer">Add answer</label>
-          <input id="add-answer" type="text" />
+          <textarea id="add-answer" type="text" name="add-answer" cols="30" rows="5"></textarea>
         </section>
-        <section>
+        <section class="create__section-wrap">
           <label for="add-tags">Add tags (divided by comma) </label>
-          <input id="add-tags" type="text" />
+          <textarea name="add-tags" id="add-tags" cols="20" rows="3"></textarea>
```

Comment:
I git the hint from Michi to use here an `<input>` in order to have a smaller box - might be an idea for you as well 😊

Reply:
first i had `<input>`, but it was to small and i liked the `<textarea>` with 3 rows more 🙂